types/foundry-vtt-api/types/framework/entities/actor.d.ts

```diff
@@ -0,0 +1,241 @@
+ *
+ * @see {@link Actors} Each Actor belongs to the Actors collection.
+ * @see {@link ActorSheet} Each Actor is edited using the ActorSheet application or a subclass thereof.
+ * @see {@link ActorDirectory} All Actors which exist in the world are rendered within the ActorDirectory sidebar tab.
+ *
+ *
+ * @example <caption>Create a new Actor</caption>
+ * let actor = await Actor.create({
+ *   name: "New Test Actor",
+ *   type: "character",
+ *   img: "artwork/character-profile.jpg",
+ *   folder: folder.data._id,
+ *   sort: 12000,
+ *   data: {},
+ *   token: {},
+ *   items: [],
+ *   flags: {}
+ * });
+ *
+ * @example <caption>Retrieve an existing Actor</caption>
+ * let actor = game.actors.get(actorId);
+ */
+declare class Actor extends Entity {
+    data: ActorData;
+
+    /**
+     * A reference to a placed Token which creates a synthetic Actor
+     */
+    token: Token;
+
+    /**
+     * Construct the Array of Item instances for the Actor
+     */
+    items: Collection<Item>;
+
+    /**
+     * Cache an Array of allowed Token images if using a wildcard path
+     */
+    protected _tokenImages: any[];
+
+    /** @override */
+    static get config(): {
+        baseEntity: Actor;
+        collection: Actors;
+        embeddedEntities: { OwnedItem: string };
+    };
+
+    /** @override */
+    prepareData(): void;
+
+    /** @override */
+    prepareEmbeddedEntities(): void;
+
+    /* -------------------------------------------- */
+    /*  Properties                                  */
+    /* -------------------------------------------- */
+
+    /**
+     * A convenient reference to the file path of the Actor's profile image
+     */
+    get img(): string;
+
+    /**
+     * A boolean flag for whether this Actor is a player-owned character.
+     * True if any User who is not a GM has ownership rights over the Actor entity.
+     */
+    get isPC(): boolean;
+
+    /**
+     * Test whether an Actor entity is a synthetic representation of a Token (if true) or a full Entity (if false)
+     */
+    get isToken(): boolean;
+
+    /**
+     * Create a synthetic Actor using a provided Token instance
+     * If the Token data is linked, return the true Actor entity
+     * If the Token data is not linked, create a synthetic Actor using the Token's actorData override
+     * @param token
+     */
+    static fromToken(token: Token): Actor;
+
+    /**
+     * Create a synthetic Token Actor instance which is used in place of an actual Actor.
+     * Cache the result in Actors.tokens.
+     * @param baseActor
+     * @param token
+     */
+    static createTokenActor(baseActor: Actor, token: Token): Actor;
+
+    /**
+     * Retrieve an Array of active tokens which represent this Actor in the current canvas Scene.
+     * If the canvas is not currently active, or there are no linked actors, the returned Array will be empty.
+     *
+     * @param linked	Only return tokens which are linked to the Actor. Default (false) is to return all
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entities/actor.d.ts;
  Line: 148;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.